kernel/uGraph.ml

```diff
@@ -274,6 +281,7 @@ let explain_universe_inconsistency prl (o,u,v,p : univ_inconsistency) =
   | Sorts.Prop -> str "Prop"
   | Sorts.SProp -> str "SProp"
   | Sorts.Type u -> Universe.pr_with prl u
+  | Sorts.QSort (_q, u) -> Universe.pr_with prl u (* FIXME? *)
```

Comment:
Fixme?

Reply:
Maybe when we give the user some control over quality variables, but for now it doesn't make sense because the kernel doesn't see them and the upper layers have their own checks around variables.

Reply:
This should respect the printing flag.

Reply:
My point above is that I think this is dead code in master.

Reply:
If it's dead code then make it assert false

Reply:
I really don't see the point of risking to have instances where this actually happens make Coq unusable when this is only used under a debug printing flag that is not even documented.

Reply:
You can document the printing flag if you want

Reply:
continued in https://github.com/coq/coq/pull/17075